packages/file-type-icons/src/FileTypeIconMap.ts

```diff
@@ -330,6 +330,7 @@ export const FileTypeIconMap: { [key: string]: { extensions?: string[] } } = {
   onetoc: {
     extensions: ['ms-one-stub', 'onetoc', 'onetoc2', 'onepkg'], // this icon represents a complete, logical notebook.
   },
+  pbiapp: {},
```

Comment:
did you mean to not have any extensions match here?

Reply:
This is a "non-classic" filetype, meaning it is not invoked by passing the file extension because it has none. Clients of the library would invoke it by passing this id as "ItemType" instead. Same situation as for the sharedfolder icon, Sway stories, Stream videos, and so on

Reply:
The PowerBI-related icon that _does_ have extensions (because it represents the content types from PowerBI that are modelled as classic files) is `powerbi`